include/quda.h

```diff
@@ -1342,6 +1347,8 @@ extern "C" {
 
   void computeKSLinkQuda(void* fatlink, void* longlink, void* ulink, void* inlink,
                          double *path_coeff, QudaGaugeParam *param);
+                         
+  void computeTwoLinkQuda(void *twolink, void *inlink, QudaGaugeParam *param);
```

Comment:
Add doxygen for this function.

Reply:
done

Reply:
Missing `in/out` on `param` descriptor here

Reply:
fixed